guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/boo-who/index.md

```diff
@@ -53,7 +53,7 @@ Uses the operator `typeof` to check if the variable is a boolean. If it is, it w
 
 #### Relevant Links
 
-*   <a>Using typeof</a>
+*   <a href ='https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/debugging/use-typeof-to-check-the-type-of-a-variable' target='_blank' rel='nofollow'>Using typeof</a>
```

Comment:
Since there is already an MDN link for **typeof**, I suggest just removing the first bullet all together
```suggestion
```

Reply:
Okay, didn't see the MDN link before. Thanks for pointing out!